infra/terraform/modules/data_access/gh_webhook_handler/index.py

```diff
@@ -0,0 +1,91 @@
+import hmac
+from hashlib import sha1
+import json
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+SNS_ARN = os.environ.get("SNS_ARN")
```

Comment:
I didn't know `os.environ` was a dictionary, I thought some kind of class you had to access via `get()`. I'll change to use `[]` so it blows up if the environment variable is not set for whatever reason.